app/assets/javascripts/components/faye_messages.jsx

```diff
@@ -0,0 +1,41 @@
+class FayeMessages extends React.Component {
+  constructor(props) {
+    super(props);
+     this.state = {
+      urlType:`/assets/chatroom.jpg`
+      };
+  }
+
+  generate_private_chat(e) {
+    var name = e.target.innerHTML;
+    if (!document.querySelector(`#${name}`)) {
```

Comment:
any good reason we can not use getElementById here as # usually means your are looking for id?